.github/workflows/tf-aws-test.yml

```diff
@@ -1,21 +1,44 @@
 name: SDK Spec AWS Tests
+run-name: SDK Spec AWS Tests (${{ inputs.repo || github.repository }}/${{inputs.ref || github.ref}})
 on:
   workflow_call: {}
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      repo:
+        type: string
+        default: winglang/wing
+      ref:
+        type: string
+        default: main
 
 env:
   AWS_REGION: "us-east-1"
   NODE_VERSION: "18.16.0"
   # this variable indicates wheater to use wing cli from a local version (using the current repo code)
   # or use the latest remote npm version
-  LOCAL_BUILD: ${{ github.workflow == 'build'}}
+  LOCAL_BUILD: ${{ github.workflow == 'build' && github.event_name == 'workflow_dispatch'}}
+  # indicates if the action triggered from a workflow_dispatch
+  MANUAL: ${{ github.event_name == 'workflow_dispatch' }}
+  REPO: ${{inputs.repo || github.repository}}
+  REF: ${{inputs.ref || github.ref}}
+  ALLOWED_USERS: ("tsuf239" "MarkMcCulloh" "eladb" "revitalbarletz" "ShaiBer" "Chriscbr" "hasanaburayyan" "staycoolcall911" "ekeren")
```

Comment:
I hope there's a better way to specify the @mainters - perhaps someone would have a concrete idea.
In any case, this list is missing names